src/v/cluster/metadata_dissemination_types.h

```diff
@@ -251,7 +254,8 @@ struct adl<cluster::ntp_leader_revision> {
         auto term = adl<model::term_id>{}.from(in);
         auto leader = adl<std::optional<model::node_id>>{}.from(in);
         auto revision = adl<model::revision_id>{}.from(in);
-        return {std::move(ntp), term, leader, revision};
+        return {
+          std::move(ntp), term, leader, revision, model::initial_revision_id{}};
```

Comment:
having trouble reconciling the changes to only the `from` part of adl encoding, and no apparent updates to the serde encoding, but the serde version is still bumped.